Client/Frontend/Settings/SettingsContentViewController.swift

```diff
@@ -59,14 +64,14 @@ class SettingsContentViewController: UIViewController, WKNavigationDelegate, The
     }
 
     // The view shown while the content is loading in the background web view.
-    fileprivate var interstitialView: UIView!
-    fileprivate var interstitialSpinnerView: UIActivityIndicatorView!
-    fileprivate var interstitialErrorView: UILabel!
+    private var interstitialView: UIView! = .build()
```

Comment:
We usually use lazy vars e.g. private lazy var interstitialView: UIView = .build()